public/javascripts/SVLabel/src/SVLabel/data/Form.js

```diff
@@ -148,8 +159,10 @@ function Form (labelContainer, missionModel, missionContainer, navigationModel,
                     });
                 }
             }
+            
+            var panorama_id = ("location" in panoData && "pano" in panoData.location) ? panoData.location.pano : "";
```

Comment:
Very tiny, but as I look at this, I want the variable to be `panoId`, both to keep it short and sweet, and to use the right casing for JS